core/templates/dev/head/domain/objects/NumberWithUnitsObjectFactory.ts

```diff
@@ -17,10 +17,12 @@
  * domain objects.
  */
 
-require('domain/objects/FractionObjectFactory.ts');
-require('domain/objects/UnitsObjectFactory.ts');
-
-require('domain/objects/objects-domain.constants.ajs.ts');
+import { Injectable } from '@angular/core';
+import { downgradeInjectable } from '@angular/upgrade/static';
+import { FractionObjectFactory } from 'domain/objects/FractionObjectFactory.ts';
```

Comment:
Add another line above this to distinguish between Angular's imports and Oppia's.